daemon/call.c

```diff
@@ -2972,7 +2972,10 @@ static int monologue_subscribe_request1(struct call_monologue *src_ml, struct ca
 
 		codec_handlers_update(dst_media, src_media, flags, sp);
 
-		MEDIA_SET(dst_media, SEND);
+		if (MEDIA_ISSET(src_media, SEND))
```

Comment:
The SEND/RECV flags are from rtpengine's POV, so I think the destination SEND should be set when the source RECV is set.

Reply:
Oh, sorry, I honestly set it empirically, and it was working fine for my use case. Was not aware of the project's workflow, but I am learning it. Will push a fix soon.